### ISSUE TITLE
Strong name signing for assembly in NuGet package

### ISSUE THREAD

**Issue by Taritsyn** [JeringTech/Javascript.NodeJS#64]
Hello, Jeremy!

Could you please sign the `Jering.Javascript.NodeJS.dll` assembly.

You must first [generate the `*.snk` file](https://docs.microsoft.com/en-us/dotnet/standard/assembly/create-public-private-key-pair), and then add the following code to the `*.csproj` file:

```xml
<Project>
  …
  <PropertyGroup>
    …
    <AssemblyOriginatorKeyFile>PATH_TO_YOUR_KEY_FILE</AssemblyOriginatorKeyFile>
    <SignAssembly>true</SignAssembly>
    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
    …
  </PropertyGroup>
  …
</Project>
```

**Comment by JeremyTCD** [comment]
Hey thanks for reminding me, was planning to sign my assemblies. Will do it.

**Comment by JeremyTCD** [comment]
@Taritsyn Good news, just released v5.3.0, strong named. If it works kindly close this issue.

**Comment by Taritsyn** [comment]
Thank you very much!
